src/App.js

```diff
@@ -0,0 +1,8 @@
+import './App.css';
+import Greeting from './components/Greeting';
+
+function App() {
+  return <Greeting />;
+}
```

Comment:
- [ ] it would be better if you add `react-router` and create a route that leads to `Greetings` in order to pass the requirements 
<img width="379" alt="Screen Shot 2022-10-04 at 17 05 41" src="https://user-images.githubusercontent.com/61790587/193869916-cba40460-2498-48bf-ac38-0a418dec8ac5.png">